README.md

```diff
@@ -2,6 +2,8 @@
 
 # Z
 
+This repo is a fork of [jethrokuan/z](https://github.com/jethrokuan/z) with support for opening file explorer on Windows.
```

Comment:
could you also exclude this line from the PR?

Reply:
Oh sure!